docs/email/faq.html

```diff
@@ -0,0 +1,156 @@
+---
+layout: page
+title: Frequently Asked Questions
+description: Answers to a few common questions.
```

Comment:
This line doesn't seem important here based on the page title. Instead I would provide a numbered list of all questions in the doc so I can quick jump to the one that provides the answer I need.